build_scripts/40-services.sh

```diff
@@ -22,6 +22,9 @@ systemctl --global enable ublue-user-setup.service
 systemctl mask bootc-fetch-apply-updates.timer bootc-fetch-apply-updates.service
 systemctl enable check-sb-key.service
 
+# Disable lastlog display on previous failed login in GDM (This makes logins slow)
+sudo authselect enable-feature with-silent-lastlog
```

Comment:
```suggestion
authselect enable-feature with-silent-lastlog
```

No need for sudo anywhere on the container I think